core/src/test/java/com/pottda/game/model/ProjectileTest.java

```diff
@@ -0,0 +1,149 @@
+package test.java.com.pottda.game.model;
+
+import com.pottda.game.model.*;
+import com.pottda.game.model.Character;
+import com.pottda.game.model.builders.AbstractModelBuilder;
+import com.pottda.game.model.builders.CharacterBuilder;
+import com.pottda.game.model.builders.ProjectileBuilder;
+import com.pottda.game.physicsBox2D.Box2DPhysicsActorFactory;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import javax.vecmath.Vector2f;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.*;
+
+public class ProjectileTest {
+    PhysicsActorFactory paf = new PhysicsActorFactory() {
```

Comment:
By this point we should maybe just make a concrete implementation in the test-folder so that we can use the same factory for each test.